dubbo-common/src/main/java/org/apache/dubbo/config/ServiceConfigBase.java

```diff
@@ -194,7 +194,7 @@ public void checkDefault() throws IllegalStateException {
         if (provider == null) {
             provider = ApplicationModel.getConfigManager()
                     .getDefaultProvider()
-                    .orElse(new ProviderConfig());
+                    .orElseGet(ProviderConfig::new);
```

Comment:
This will put the ProviderConfig instance into ConfigManager, which is not an action expected.